### PR TITLE
Added comments

### PR DESCRIPTION
**Pull request by Luke2v52**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -10,6 +12,10 @@ def index
   
     # GET /users/1 or /users/1.json
     def show
+      # if the user being shown has a committee then this redirects to the current members page
+      # otherwise they will be sent to the pending members page
+      # this is just a workaround to prevent this show method from being used without
+      # removing its potential future use
```
Comment (Buzzec) [inline app/controllers/users_controller.rb:18]: Put above method

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -19,8 +25,9 @@ def show
   
     # GET /users/new
     def new
+      # redirect people to the events page if they are trying 
       redirect_to root_path
-      @user = User.new
+      #@user = User.new this controller should not make a new user as users should be either seeded or made by sign up
```
Comment (Buzzec) [inline app/controllers/users_controller.rb:30]: Remove dead code

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (Buzzec) [inline app/controllers/users_controller.rb:18]: Put above method

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -19,8 +25,9 @@ def show
   
     # GET /users/new
     def new
+      # redirect people to the events page if they are trying
```
Comment (Buzzec) [inline app/controllers/users_controller.rb:28]: Put above method

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -29,22 +36,24 @@ def edit
...
-      end
+      # respond_to do |format|
+      #   if @user.save
+      #     format.html { redirect_to @user, notice: "user was successfully created." }
+      #     format.json { render :show, status: :created, location: @user }
+      #   else
+      #     format.html { render :new, status: :unprocessable_entity }
+      #     format.json { render json: @user.errors, status: :unprocessable_entity }
+      #   end
+      # end
```
Comment (Buzzec) [inline app/controllers/users_controller.rb:50]: Remove dead code

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -29,22 +36,24 @@ def edit
   
     # POST /users or /users.json
     def create
-      @user = User.new(user_params)
+      # this method should never be called for the members pages
+      # @user = User.new(user_params)
```
Comment (Buzzec) [inline app/controllers/users_controller.rb:40]: Remove dead code

**Review by Buzzec — DISMISSED** [review]

**Review by sumedhtadimeti — APPROVED** [review]
Looks goo!

Thanks for following good coding standards

**Pull request merged**